### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by lethargistic** [junguler/m3u-radio-music-playlists#5] [General]
Just wanted to say thank you cause that's just what I was searching for.

**Comment by junguler** [comment]
you are welcome friend

**Comment by chessgpt** [comment]
I can only add to that. Thank you so much for this incredible collection and taking the time to keep it up to date.

I am using this in Kodi.

**Comment by junguler** [comment]
you are welcome friend, means a lot
